tests/test_e2e.py

```diff
@@ -20,17 +20,20 @@ def test_pdf_extractions(context_dir, http_mock, client, worker, gen_tds_artifac
     text = ""
     for d in text_json:
         text += f"{d['content']}\n"
-    tds_artifact = gen_tds_artifact()
-    tds_artifact["file_names"] = ["paper.pdf"]
-    tds_artifact["metadata"] = {"text": text}
+    tds_artifact = gen_tds_artifact(
```

Comment:
I wonder if we should consider renaming this stuff from `tds_artifact` since we don't use `artifacts` at all in KM anymore, just `code` and `documents`?